### PR TITLE
fix: fluent-bit patched image with fixed CVE

### PR DESCRIPTION
**Pull request by SandhyaRavi2403**
**What problem does this PR solve?**:
Patch fluent-bit image with fixed CVE's

workflow link:
https://github.com/mesosphere/dkp-container-images/actions/runs/9929694662

**Which issue(s) does this PR fix?**:

https://jira.nutanix.com/browse/NCN-101394

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] If the PR adds a version bump, ensure there is no breaking change in Licensing model (or NA).
- [ ] If a chart is changed or app configuration is significantly changed, the chart version is correctly incremented (so that apps are not automatically upgraded from a previous version of DKP).

### CODE REVIEW

**Review by mhrabovcin — APPROVED** [review]

File: services/fluent-bit/0.46.9-d2iq.0/defaults/cm.yaml
Comment (mhrabovcin) [inline services/fluent-bit/0.46.9-d2iq.0/defaults/cm.yaml:1]: https://github.com/mesosphere/kommander-applications/blob/main/services/fluent-bit/0.46.9/defaults/cm.yaml#L4

I think that this CM name needs to be removed to match the  service name and indluce the `-d2iq.0` in the name.

Comment (SandhyaRavi2403) [reply services/fluent-bit/0.46.9-d2iq.0/defaults/cm.yaml:1]: Updated

**Comment by mhrabovcin** [comment]
@SandhyaRavi2403 I believe we can merge this one

**Comment by SandhyaRavi2403** [comment]
> @SandhyaRavi2403 I believe we can merge this one

Yes. 
The corresponding Kommander PR _https://github.com/mesosphere/kommander/pull/4797_,  [Kuttl Kommander Operator Install](https://github.com/mesosphere/kommander-applications/pull/2406#logs) is failing. Is it necessary to wait for this test to turn green.

**Comment by mhrabovcin** [comment]
@SandhyaRavi2403 that check isn't mandatory and it is now broken so we can ignore it for now.

**Pull request merged**